src/lib/entities/entry.ts

```diff
@@ -8,13 +8,15 @@ class Entry {
   private _version: number
   private _fields: object
   private _publishedVersion?: number
+  private _tags?: object[]
```

Comment:
Ah it was defined just below 🙂.
Anyway, same concerns. I think it might be fine to keep `tags` as a top level key since the implementation details are hidden from the users, but eventually in the future we might want to nest this under a `Metadata` object if we introduce more things there.
Still I'd say we should use explicit typings here instead of `object[]`.

Reply:
I agree! Using tags was just the most straightforward solution for now, and I will update the typings.